lib/fizzbuzz.rb

```diff
@@ -2,10 +2,19 @@ class SuperFizzBuzz
 
   def run(input)
 
-    #Implement your code here
-
+    if input.modulo(3).zero?
```

Comment:
BTW, I much prefer the way you wrote this `input.modulo(3).zero?` instead of what most folks write: `input % 3 == 0`.